tcms/report/views.py

```diff
@@ -81,71 +82,83 @@ def overview(request, product_id, template_name='report/overview.html'):
 
 class ProductVersionReport(TemplateView, ProductVersionReportData):
     template_name = 'report/version.html'
+    product = None
+    selected_version = None
 
-    def get(self, request, product_id):
+    def get(self, request, product_id):  # pylint: disable=arguments-differ
```

Comment:
Why the disable pragma? What is causing the error ?

Reply:
I disabled it due to a misunderstanding on my side. I'll fix it right away.